packages/electron-commons/test/initializers/electron-node.spec.ts

```diff
@@ -41,15 +44,31 @@ describe('onDisconnectHandler for node environment initializer', () => {
         errorMessage: '',
     };
 
+    const expectedDisposeResult: ProcessExitDetails = {
+        exitCode: 0,
+        signal: null,
+        errorMessage: '',
```

Comment:
errorMessage should not be defined when there was no error
also, it should not default to an empty string. most stuff shouldn't.

Reply:
done